tests/foreman/ui_airgun/test_os.py

```diff
@@ -82,9 +79,10 @@ def test_positive_delete(session):
     major = gen_string('numeric', 2)
     entities.OperatingSystem(name=name, major=major).create()
     with session:
-        assert session.operatingsystem.search(name) == '{} {}'.format(name, major)
+        assert session.operatingsystem.search(name) == '{} {}'.format(
+            name, major)
         session.operatingsystem.delete(name)
-        with pytest.raises(NoSuchElementException):
-            session.operatingsystem.search(name)
-    current = entities.OperatingSystem().search(query={'search': 'name="%s"' % name})
+        assert session.operatingsystem.search(name) == None
+    current = entities.OperatingSystem().search(query={
```

Comment:
seriously, remove that. We are testing webUI here only, even if all application API will be ruined in result - we don't care